boilerplate/app/screens/demo/demo-list-screen.tsx

```diff
@@ -41,46 +42,48 @@ const FLAT_LIST: ViewStyle = {
   paddingHorizontal: spacing[4],
 }
 
-export const DemoListScreen = observer(function DemoListScreen() {
-  const navigation = useNavigation()
-  const goBack = () => navigation.goBack()
+export const DemoListScreen: FC<StackScreenProps<NavigatorParamList, "demoList">> = observer(
+  ({ navigation }) => {
```

Comment:
I've merged this, but just as a question, why switch to using props to grab the navigation reference rather than using the `useNavigation` hook? I kinda prefer the hook from an aesthetic standpoint.

Reply:
@jamonholmgren it's because I wasn't sure how to type the `useNavigation` hook. Maybe it takes a generic.

Reply:
Ah! Good answer. Thanks.